clusters/app.ci/vault/manifests.yaml

```diff
@@ -0,0 +1,175 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: vault
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: cert-issuer
+    route.openshift.io/termination: passthrough
+  name: vault
+  namespace: vault
+spec:
+  tls:
+    - secretName: vault-tls
+      hosts:
+        - vault.ci.openshift.org
+  rules:
+  - host: vault.ci.openshift.org
+    http:
+      paths:
+      - backend:
+          service:
+            name: vault
+            port:
+              number: 8200
+        pathType: ImplementationSpecific
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: vault
+  namespace: vault
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - name: 1.6.2
+    from:
+      kind: DockerImage
+      # Mirror of docker.io/library/vault:1.6.2 because the cluster hit the ratelimit for that
+      name: quay.io/alvaroaleman/vault:1.6.2
```

Comment:
Why not just maintain the Dockerfile instead?

Reply:
Because its more work to maintain a dockerfile than to pull and push

Reply:
Well, in the future if this image has to change or we need to debug something related to the image, I would prefer to have it in our cluster. I agree that it is more work to maintain the dockerfile but I can live with that. Your call.

Reply:
It is in our cluster, that is the purpose of the manifest. The cluster just happened to exceed the quota for dockerhub yesterday, so I pushed it locally to quay. The next time we update this image, we can just change the name to a dockerhub reference and be done.